walkthroughs/http-headers-and-priority/k8s/deploy.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -e
+
+if [ -z "${AWS_ACCOUNT_ID}" ]; then
+    echo "AWS_ACCOUNT_ID must be set."
+    exit 1
+fi
+
+if [ -z "${AWS_DEFAULT_REGION}" ]; then
+    echo "AWS_DEFAULT_REGION must be set."
+    exit 1
+fi
+
+if [ -z "${MESH_NAME}" ]; then
```

Comment:
Should this be EKS_CLUSTER not MESH_NAME?

Reply:
MESH_NAME is more appropriate given that this example should work outside of EKS, i.e. any K8s cluster